ibm/service/database/resource_ibm_database_mongodb_sharding_test.go

```diff
@@ -138,8 +138,8 @@ func testAccCheckIBMDatabaseInstanceMongoDBShardingFullyspecified(databaseResour
 		}
 		users {
 		  name     = "user124"
-		  password = "password12password"
-		  type     = "database"
+		  password = "password$12password"
+		  type     = "ops_manager"
```

Comment:
Has this been tested with MongoDBEE? Could you please add a test to cover that as well?